### PR TITLE
Update theming to correctly match ISTEX design

### PR DESCRIPTION
**Pull request by AlasDiablo**

### CODE REVIEW

File: tdm-fe/src/app/components/form/ProcessingFormEmail.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ const ProcessingFormEmail = () => {
             />
             {isInvalid ? (
                 <div className="text processing-form-field-label error">
-                    L&apos;adresse électronique n&apos;est pas valide
+                    L&lsquo;adresse électronique n&apos;est pas valide
```
Comment (parmentf) [inline tdm-fe/src/app/components/form/ProcessingFormEmail.tsx:28]:
```suggestion
                    L&apos;adresse électronique n&apos;est pas valide
```

File: tdm-fe/src/app/components/form/ProcessingFormUpload.tsx [hunk 2]
```diff
@@ -52,7 +52,7 @@ const ProcessingFormUpload = () => {
                         <>Fichier manquant</>
                     ) : (
                         <>
-                            Le fichier ne correspond pas a un format compatible, utilisé l&apos;un de ces format :{' '}
+                            Le fichier ne correspond pas a un format compatible, utilisé l&lsquo;un de ces format :{' '}
```
Comment (parmentf) [inline tdm-fe/src/app/components/form/ProcessingFormUpload.tsx:55]:
```suggestion
                            Le fichier ne correspond pas à un format compatible, utilisez l&apos;un de ces formats :{' '}
```

File: tdm-fe/src/app/components/layout/WebServicesFooter.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import Container from '@mui/material/Container';
+
+const WebServicesFooter = () => {
+    return (
+        <div id="ws-footer">
+            <Container>
+                <p id="ws-footer-galaxy">La galaxie Istex</p>
+                <h1>Découvrez tous nos web services</h1>
+                <p id="ws-footer-text">
+                    L&lsquo;équipe Istex vous propose une pluralité d&lsquo;outils pour répondre à vos besoins
```
Comment (parmentf) [inline tdm-fe/src/app/components/layout/WebServicesFooter.tsx:12]:
```suggestion
                    L&apos;équipe Istex vous propose une pluralité d&apos;outils pour répondre à vos besoins
```

File: tdm-fe/src/app/components/layout/WebServicesHeader.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import '~/app/components/layout/scss/WebServicesHeader.scss';
+import Container from '@mui/material/Container';
+
+const WebServicesHeader = () => {
+    // Todo add the correcte text when available
+    return (
+        <div id="ws-header">
+            <Container>
+                <p className="text">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sit amet vulputate ex. Vivamus vitae
```
Comment (parmentf) [inline tdm-fe/src/app/components/layout/WebServicesHeader.tsx:10]: **nitpick (non-blocking):** ‏There should be a real sentence. :P

File: tdm-fe/src/app/components/progress/FileUpload.tsx [hunk 5]
```diff
@@ -11,7 +11,7 @@ const FileUpload = ({ showError }: FileUploadProps) => {
         return <CircularWaiting />;
     }
 
-    return <Alert severity="error">Le fichier n&apos;a pas pu être téléversé.</Alert>;
+    return <Alert severity="error">Le fichier n&lsquo;a pas pu être téléversé.</Alert>;
```
Comment (parmentf) [inline tdm-fe/src/app/components/progress/FileUpload.tsx:14]:
```suggestion
    return <Alert severity="error">Le fichier n&apos;a pas pu être téléversé.</Alert>;
```

**Review by parmentf — CHANGES_REQUESTED** [review]

File: tdm-fe/src/app/components/form/ProcessingFormStepper.tsx [hunk 6]
```diff
@@ -35,27 +36,48 @@ const steps = [
...
+        if (isExtended && (isFirst || isLast)) {
+            left = '8px';
+        }
+
+        return styled(StepConnector)(({ theme }) => ({
+            [`&.${stepConnectorClasses.alternativeLabel}`]: {
+                top: 16,
+                left: left,
+                right: 'calc(50% + 18px)',
+                transform: isExtended && isLast ? 'translateX(calc(100% + 36px))' : undefined,
```
[the remark this answers is not in the document]
Comment (AlasDiablo) [reply tdm-fe/src/app/components/form/ProcessingFormStepper.tsx:60]: This is to implement the right bar on the stepper, and I needed a trick like this to allow advanced theming.

File: tdm-fe/src/app/components/layout/WebServicesHeader.tsx (hunk 4 above)
Comment (AlasDiablo) [reply tdm-fe/src/app/components/layout/WebServicesHeader.tsx:10]: Communication departments are working on this.

File: tdm-fe/src/app/components/form/ProcessingFormStepper.tsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (parmentf) [reply tdm-fe/src/app/components/form/ProcessingFormStepper.tsx:60]: Oh. Translate X. Not trans LaTeX.

**Review by parmentf — APPROVED** [review]

**Pull request merged**